spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/StreamControllerTests.java

```diff
@@ -178,6 +178,96 @@ public void testSave() throws Exception {
 	}
 
 	@Test
+	public void testFindRelatedStreams() throws Exception {
+		assertEquals(0, repository.count());
+		mockMvc.perform(
+				post("/streams/definitions/").param("name", "myStream1").param("definition", "time | log")
+						.accept(MediaType.APPLICATION_JSON)).andDo(print())
```

Comment:
Please remove the `andDo(print())`s, we have enough garbage in the console while building already.

Also, I think there is the way to have `accept(MediaType.APPLICATION_JSON)` as a default for the whole test